docs/how_to_add_packages.md

```diff
@@ -132,6 +134,13 @@ Then in your `conanfile.py` method, it has to be used to download the sources:
      tools.get(**self.conan_data["sources"][self.version])
 ```
 
+### How to provide a good recipe
+
+The [recipes](https://github.com/conan-io/conan-center-index/tree/master/recipes) available in CCI can be used as good examples, you can use them as the base for your recipe.
+
+If you are looking for header-only projects, you can take a look on [rapidjson](https://github.com/conan-io/conan-center-index/blob/master/recipes/rapidjson/all/conanfile.py), [rapidxml](https://github.com/conan-io/conan-center-index/blob/master/recipes/rapidxml/all/conanfile.py), and [nuklear](https://github.com/conan-io/conan-center-index/blob/master/recipes/nuklear/all/conanfile.py). Also, Conan Docs has a section about [how to package header-only libraries](https://docs.conan.io/en/latest/howtos/header_only.html).
+
+For C/C++ projects which use CMake for building, you can take a look on [szip](https://github.com/conan-io/conan-center-index/blob/master/recipes/szip/all/conanfile.py) and [recastnavigation](https://github.com/conan-io/conan-center-index/blob/master/recipes/recastnavigation/all/conanfile.py).
```

Comment:
Maybe add exotic recipes, like proxy package?

Reply:
I would rather concentrate on common cases to help people who aren't yet too familiar with Conan and/or CCI.

If the documentation has information about everything, it is harder for a beginner to filter out the important parts for their specific goal

Reply:
Right, let's keep it simple!

Reply:
I think that you can also add one example relying on autotools.